django_elasticsearch_dsl/documents.py

```diff
@@ -63,29 +63,37 @@ def _format_index_language(cls, index, language):
         return '{0}-{1}'.format(index, language)
 
     @classmethod
-    def get_index_name_for_language(cls, index, language):
+    def get_custom_index_name(cls, index, language=None):
+        """
+        Helper function to fetch the index name at this env
+        """
+        language_dsl_enabled = getattr(settings, 'ELASTICSEARCH_DSL_TRANSLATION_ENABLED', False)
+        language = language if language else settings.LANGUAGE_ENGLISH
```

Comment:
Can be simplified as
```suggestion
        language = language or settings.LANGUAGE_ENGLISH
```